src/Images.css

```diff
@@ -16,3 +16,11 @@
     --pf-c-table__action--PaddingTop: 0.5rem;
     --pf-c-table__action--PaddingBottom: 0.5rem;
 }
+
+#containers-images .pf-c-table.pf-m-compact .pf-c-table__toggle {
```

Comment:
This will create inconsistent left padding between the images and containers toggles in lists. 
Could you please make it apply also for Containers and move it to podman.scss file?

Reply:
Something like this I mean:

``` diff
diff --git src/Images.css src/Images.css
index 3caed167..f080943f 100644
--- src/Images.css
+++ src/Images.css
@@ -17,10 +17,6 @@
     --pf-c-table__action--PaddingBottom: 0.5rem;
 }
 
-#containers-images .pf-c-table.pf-m-compact .pf-c-table__toggle {
-    padding-left: 0px;
-}
-
 .images-grey-text {
    color: var(--pf-global--Color--200);
 }
diff --git src/podman.scss src/podman.scss
index 43aa363f..d1ad85af 100644
--- src/podman.scss
+++ src/podman.scss
@@ -8,6 +8,12 @@
     @extend .ct-card;
 }
 
+#containers-images, #containers-containers {
+    .pf-c-table.pf-m-compact .pf-c-table__toggle {
+        padding-left: 0px;
+    }
+}
+
 .container-block {
     display: flex;
     flex-direction: column;

```

Reply:
And last thing - because we are all not really good with CSS I like adding comment so that we don't get lost. 
In this case, decrease default padding for the toggle button in list.